src/gta/script_handler.hpp

```diff
@@ -255,9 +266,51 @@ class CGameScriptHandlerNetComponent
 	virtual bool _0x118() = 0; // related to above function
 
 	CGameScriptHandler* m_script_handler; //0x0008
-	char pad_0010[48]; //0x0010
-	std::uint32_t m_participants; //0x0040
-};
+	char pad_0010[32]; //0x0010
+	class CScriptParticipant* m_host; //0x0030
+	int16_t m_local_participant_index; //0x0038
+	char pad_003A[6]; //0x003A
+	uint32_t m_participant_bitset; //0x0040
+	char pad_0044[36]; //0x0044
+	class CScriptParticipant* m_participants[32]; //0x0068
+	char pad_0168[12]; //0x0168
+	int16_t m_num_participants; //0x0174
+	char pad_0176[28]; //0x0176
+	uint8_t m_host_migration_flags; //0x0192
+	char pad_0193[29]; //0x0193
+
+	// does not work on local player
+	int get_participant_index(CNetGamePlayer* player);
```

Comment:
Maybe i'm stupid but `src/gta/script_handler.cpp` seems to have code for local player.